File: java/hotel_agadir/Manager.java
package hotel_agadir;
import java.util.Scanner;

public class Manager {
    
    private Room_type new_room;
    public Manager(Room_type new_room){
        this.new_room=new_room;
    }
    public Room_type get_rm(){
        return new_room;
    }
    public void set_rm(Room_type new_room){
        this.new_room=new_room;
    }

    public void hotel_organize(){
    
    Scanner scan=new Scanner(System.in);
    Room constr=new Room();
    
    System.out.println("Do you want to add a new room or to update a room booking information? \n");
    System.out.println("    1)Add a new room to the system");
    System.out.println("    2)Modify book information");
    int choice=scan.nextInt();
    scan.nextLine();
    if(choice==1){
        System.out.println("What's the number of rooms you want to enter the infomation:");
        int x=scan.nextInt();
        scan.nextLine();
       
        for(int i=0; i<x; i++){
        System.out.println("Enter the following information\n");
        System.out.println("room size:\n");
        new_room.room_size=scan.nextLine();
        System.out.println("room number:\n");
        new_room.room_num=scan.nextInt();
        System.out.println("room floor:\n");
        new_room.floor_num=scan.nextInt();
        System.out.println("reservation status:\n");
        new_room.booked=scan.nextBoolean();
        constr.enter_rooms(new_room);
        scan.nextLine();
        }
    }
    else if(choice==2){
        System.out.println("enter room number:");
        CharSequence x=scan.nextLine();
        System.out.println("enter reservation status");
        Boolean y=scan.nextBoolean();
        constr.edit(x,y);
        
    }

    }
    
}

File: hotel_agadir/Manager.js
const fs=require('fs');
const Room=require('./Room');
const prompt=require('prompt-sync')({sigint: true})
class Manager{
    constructor(new_room){
        this.new_room=new_room;
    }
    hotel_organize(){
        const room=new Room();
        let new_room={size:'null',number:0,floor_number:0, booking_status:false};
        console.log("Do you want to add a new room or to update a room booking information? \n");
        let choice=prompt("1)Add a new room to the system 2)Modify book information\n");

        if(choice===1){
            let x=prompt("What's the number of rooms you want to enter the infomation:\n");
            for(let i=0;i<x;i++){
                console.log("Enter the following information\n");
                new_room.size=prompt("room size:\n");
                new_room.number=prompt("room number:\n");
                new_room.floor_number=prompt("floor number:\n");
                new_room.booked=prompt('reservation status:\n');
                room.enter_rooms(new_room);
            }
        }
        else if(choice===2){
            let x=prompt("Enter room number: ");
            let y=prompt("Enter reservation status: ");
            room.edit(x,y);
        }
    }
}
module.exports=Manager;